aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/MutationProcessor.java

```diff
@@ -72,8 +71,11 @@
      * it again later, when network conditions become favorable again.
      */
     void startDrainingMutationOutbox() {
-        disposable.add(
+        ongoingOperationsDispoable.add(
             mutationOutbox.observe()
+                .doOnSubscribe(disposable ->
+                    LOG.info("Started processing the mutation outbox. Pending mutations will be published to cloud.")
```

Comment:
![](https://i.pinimg.com/564x/12/36/5f/12365fc27db1a2820559cf24d5bd5fa4.jpg)

Sorry, I can't help it.

Reply:
Haha -- what is this guys' name again?

Fixed, to "the cloud". :-D

Reply:
I believe that’s Mr. McGibblets. https://youtu.be/1VOb1I5K6bg

Reply:
Right!!! McGibblets :-)